deploy/helm/distributed-compute-operator/templates/http-timeout-envoyfilter.yaml

```
@@ -0,0 +1,46 @@
+{{- if and (.Values.istio.enabled) (.Values.istio.httpIdleTimeout.timeout)  }}
+{{- $justRootConfigNamespace := list .Values.istio.rootConfigNamespace }}
+{{- $defaultNamespaces := .Values.config.watchNamespaces | default $justRootConfigNamespace }}
+{{- $namespaces := .Values.istio.httpIdleTimeout.namespaces | default $defaultNamespaces }}
+{{- range $namespace := $namespaces }}
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: distributed-compute-cluster-envoyfilter
```

```suggestion
  name: {{ include "common.names.fullname" . }}-envoyfilter
```

This name should be unique to prevent collisions if there are multiple releases of DCO. Unlikely, yet possible.

---

done